lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -131,7 +131,7 @@ class FirstMeaningfulPaint extends Audit {
     score = Math.min(100, score);
     score = Math.max(0, score);
 
-    timings.navStart = data.navStart.ts / 1000;
+    timings.navStart = data.navStart && data.navStart.ts / 1000;
```

Comment:
yeah, better to complain until we handle the general case of no `navStart` in a trace in #519